source/accessibility.html.md.erb

```diff
@@ -0,0 +1,120 @@
+- [Add corporate card fees](https://docs.payments.service.gov.uk/corporate_card_surcharges/#add-corporate-card-fees)
+- [Block prepaid cards](https://docs.payments.service.gov.uk/block_prepaid_cards/#block-prepaid-cards)
+- [Go live](https://docs.payments.service.gov.uk/switching_to_live/#go-live)
+- [Connect your live account to ePDQ](https://docs.payments.service.gov.uk/switching_to_live/set_up_a_live_epdq_account/#connect-your-live-account-to-epdq)
+- [Connect your live account to Smartpay](https://docs.payments.service.gov.uk/switching_to_live/set_up_a_live_smartpay_account/#connect-your-live-account-to-smartpay)
+- [Support](https://docs.payments.service.gov.uk/support_contact_and_more_information/#support)
+
+This is not a fail, but should be looked at under [WCAG 2.1 success criterion 2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html).
+
+Also, some parts of this website are not fully accessible because of [issues caused by our Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/accessibility#using-the-technical-documentation-template-for-your-own-documentation).
+
+### Feedback and contact information
+
+If you need any part of this service in a different format like large print, audio recording or braille, email [govuk-pay-support@digital.cabinet-office.gov.uk](mailto:govuk-pay-support@digital.cabinet-office.gov.uk).
+
+We’ll consider your request and get back to you within 2 working days.
+
+### Reporting accessibility problems with this website
+
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, email [govuk-pay-support@digital.cabinet-office.gov.uk](mailto:govuk-pay-support@digital.cabinet-office.gov.uk).
+
+### Enforcement procedure
+
+The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018
+(the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, [contact the Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/).
+
+## Technical information about this website’s accessibility
+
+GDS is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+
+## Compliance status
+
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard, due to the non-compliances listed below.
+
+### Non-accessible content
+
+The content listed below is non-accessible for the following reasons.
+
+#### Non-compliance with the accessibility regulations
+
+The following pages have redundant links:
```

Comment:
this seems to repeat the section above - do we need it twice?

Reply:
Yes, unfortunately, that's the template. I know it's a bit ridiculous.

Reply:
Can we push back on the template...?

Reply:
I'll ask, but I cannot make any promises :-)

Reply:
Ok, so it looks like the first section is more of a summary, so i'll redraft that statement and update the PR